vtexdocs/RapiDoc
# Feat/download openapi

#### What is the purpose of this pull request?

To add buttons to be able to view and download the open api spec.

#### How should this be manually tested?

Open the preview and check if you are able to view and download the open api spec in the API Reference pages.

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/62757720/222742353-0f8ae1cc-c6e0-4374-90ed-fbd2a206b347.png)

#### Types of changes
